guides/v15-mdc-migration.md

```diff
@@ -0,0 +1,571 @@
+
+### Select
+
+* MatSelect no longer aligns the selected option in the dropdown menu with the trigger text.
+
+* Long options now wrap by default instead of truncating.
+
+* Previously the dropdown menu width could be wider than the parent form-field, but now the dropdown
+  is the same width as the form-field
+
+### Slide Toggle
+
+* Accessibility: MDC-based version uses `<button role="switch">` to represent the toggle rather than
+  `<input type="checkbox">`
+
+* Accessibility: The touch target is much larger than the slide toggle; to match the legacy size,
+  you can provide a `-5` density to the slide toggle’s theme mixin:
+
+  ```scss
+  @use '@angular/material' as mat;
+
+  @include mat.slide-toggle-theme(
+    map-merge($theme, (density: -5))
+  );
+  ```
+
+  Without the density mixin, the touch targets collide (see blue outlined targets below) collide:
+
+  TODO(mmalerba): replace this with a working image or embedded example:
+  ![alt_text](../images/slide-toggle-diff.png "Diagram showing overlapping slide-toggle touch targets")
+
+* The label is closer to the enabled toggle
+
+### Slider
+
+* Accessibility: sliders now work with mobile device screen readers.
+
+* To accommodate range sliders, the implementation has changed from the `<mat-slider>` element being
+  the form control to the `<mat-slider>` element containing 1-2 `<input>` elements (the slider
+  "thumbs") that act as the form control(s). The value, associated events (`input`, `change`), and
+  labels (`aria-label`) now live on the `<input>` elements instead.
+
+* Vertical sliders and inverted sliders are no longer supported, as they are not part of the
+  Material Design spec.
+
+* Range sliders are now supported. TODO(mmalerba): add more about this.
+
+### Snack Bar
+
+* For simple, text-based snack-bars, there are no significant changes.
+
+* For simple snack-bars with an action button, they use the MDC-based mat-button, so your app will
+  need to include the theme sass for the MDC-based button
+
+* For snack-bars that use custom structured content (i.e. calls to `MatSnackBar.openFromComponent`
+  and `MatSnackBar.openFromTemplate`), you  should use the following new directives to annotate your
+  content:
+  * `matSnackBarLabel` to mark the text displayed to users
+  * `matSnackBarActions` to mark the element containing the action buttons
+  * `matSnackBarAction` to mark individual action buttons
+  * If you do not specify any of these directives, it will treat the entire custom component /
+    template as text.
+
+* Tests that open a snack-bar now require calling `flush()` before attempting to access the content
+  of the snackbar.
+
+### Table
+
+* Every cell has `16px` left and right padding. Previously only a `24px` padding applied to the
+  leftmost and rightmost cells.
+
+* Header text size and color matches the data rows.
+
+* Text will not wrap by default; to enable text wrapping, apply the `white-space: normal` style to
+  the `<mat-cell>`.
```

Comment:
```suggestion
* Text will not wrap by default; to enable text wrapping, apply the `white-space: normal` style to
  the table cell.
```
or is it only `<mat-cell>` and not `<td mat-cell>` ?

Reply:
both I believe